librocksdb_sys/crocksdb/c.cc

```diff
@@ -4461,6 +4461,17 @@ void crocksdb_ingestexternalfileoptions_set_allow_blocking_flush(
   opt->rep.allow_blocking_flush = allow_blocking_flush;
 }
 
+unsigned char crocksdb_ingestexternalfileoptions_get_write_global_seqno(
```

Comment:
Considering other fields don't provide getter, let's remove it to keep API consistent. But I appreciate your effort in examing it.

Reply:
I think an additional getter is fine. I think we should have getters for all fields, but we don't need to add them now.

Reply:
Ack

Reply:
@yiwu-arbug I see no case for using getters of option.